hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/ha/TestObserverReadProxyProvider.java

```diff
@@ -367,9 +366,9 @@ public void testGetHAServiceStateWithTimeout() throws Exception {
 
     setupProxyProvider(1, NAMENODE_HA_STATE_PROBE_TIMEOUT_SHORT);
     final HAServiceState state = HAServiceState.STANDBY;
-    NNProxyInfo<ClientProtocol> dummyNNProxyInfo =
+    @SuppressWarnings("unchecked") NNProxyInfo<ClientProtocol> dummyNNProxyInfo =
```

Comment:
We usually put it in a separate line. Does that work here too?

Reply:
fixed.